### PR TITLE
Update to python ^3 and Django ^3 & remove support for older versions

### PR DESCRIPTION
**Pull request by wiltso**

### CODE REVIEW

**Comment by akx** [comment]
FWIW, you may find some of this work done in https://github.com/kcsry/django-form-designer already...

**Comment by suutari-ai** [comment]
I think we should either use the kcsry's version directly or if it doesn't have all of the needed fixen, then it would be better to contribute them there. AFAIK that's the most well maintained fork of django-form-designer atm.

**Comment by akx** [comment]
Well, we have removed Django-CMS support in that fork, which I would imagine you need 😅

**Comment by suutari-ai** [comment]
Oh, OK. Then we might still need to maintain our own fork after all. :cry:

File: form_designer/email.py [hunk 1]
```diff
@@ -55,24 +53,23 @@ def build_form_mail(form_definition, form, files=None):
...
-        'body': message,
-        'from_email': from_email,
-        'to': mail_to,
+        "subject": mail_subject,
+        "body": message,
+        "from_email": from_email,
+        "to": mail_to,
     }
 
     if DJANGO_18:  # the reply_to kwarg is only supported in Django 1.8+ . . .
```
Comment (Niilis) [inline form_designer/email.py]: This check seems awfully outdated.

It should be OK to remove the if check here and the fallback a few lines down.

**Review by Niilis — CHANGES_REQUESTED** [review]
Nice to get standardized coding practices into use.

I spotted one very old check that could be removed. Also, the main README has some old claims that this commit makes faulty. It should be updated to state that this will be compatible with Django >3.

**Review by Niilis — APPROVED** [review]
I think this is good to go! :shipit:

**Pull request merged**